jep/1/README.adoc

```diff
@@ -1014,31 +1058,39 @@ but that existing RFC was a good justification for keeping the terms.
 
 === Limits of BDFL model
 
-People expressed concern over the limits of the current model where the BDFL
+During the original discussion of the JEP process in 2017,
```

Comment:
I'd prefer to make a new heading for the switch from BDFL and leave this section mostly unchanged.